cellengine/resources/experiment.py

```diff
@@ -209,28 +210,46 @@ def undelete(self):
             del self.deleted
 
     @property
-    def active_compensation(self) -> Optional[Union[Compensation, int]]:
+    def active_compensation(self) -> Optional[Union[str, int]]:
+        """The most recently used compensation.
+
+        Accepted (case-insensitive) values are one of:
+            - A Compensation object,
+            - A Compensation ID,
+            - A `cellengine` module variable, one of: [UNCOMPENSATED,
+                FILE_INTERNAL, PER_FILE]
+            - One of [0, -1, -2]
+
+        Example:
+            ```python
+            import cellengine
+            exp = cellengine.get_experiment(name='my experiment')
+            experiment.active_compensation = cellengine.UNCOMPENSATED
+            ```
+        """
         active_comp = self._active_comp
-        if type(active_comp) is str:
-            return ce.APIClient().get_compensation(self._id, active_comp)
-        elif type(active_comp) is int:
+        if isinstance(active_comp, (str, int)):
             return active_comp
         elif active_comp is None:
             return None
-        raise ValueError(
-            f"Value '{active_comp}' is not a valid for activeCompensation."
-        )
+        else:
+            raise ValueError(
+                f"Value '{active_comp}' is not valid for active_compensation."
+            )
 
     @active_compensation.setter
-    def active_compensation(self, compensation: Union[Compensation, int]):
-        if type(compensation) is Compensation:
+    def active_compensation(self, compensation: Union[Compensation, int, str]):
+        """Set the active compensation."""
```

Comment:
Are `.setter` docstrings rendered anywhere?

Reply:
Nope; should I remove it?

Reply:
they're trumped by the getter docstring